lib/cocina/models/mapping/to_mods/geographic.rb

```diff
@@ -70,12 +70,10 @@ def extract_format(geo)
           end
 
           def extract_type(geo)
-            type = geo[:form].find do |form|
-              form[:type].match(TYPE_REGEX) || (form[:type].match(MEDIA_REGEX) && form[:value] == 'Image')
+            type = geo.form.find do |form|
+              form.type.match(TYPE_REGEX) || (form.type.match(MEDIA_REGEX) && form.value == 'Image')
```

Comment:
Since we're not using the return value other than as a boolean, can we use `#match?` instead of `#match` here?

Reply:
Thanks. Fixed.